code/defines/obj/weapon.dm

```diff
@@ -735,6 +735,17 @@
 	melt_temperature=MELTPOINT_STEEL
 	flags = FPRINT
 	siemens_coefficient = 1
+	
+/obj/item/weapon/table_parts/glass
+	name = "glass table parts"
+	desc = "Glass table parts for the spaceman with style."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "glass_tableparts"
+	starting_materials = list(MAT_GLASS = 3750)
+	w_type = RECYK_GLASS
+	melt_temperature=MELTPOINT_STEEL
+	flags = FPRINT
+	siemens_coefficient = 1
```

Comment:
What, why?

I should also mention that due to being glass it should conduct less. (unless glass conducts nicely which I doubt)